libraries/AP_OpenDroneID/AP_OpenDroneID.cpp

```diff
@@ -259,7 +260,20 @@ void AP_OpenDroneID::send_location_message()
     if (!ahrs.get_position(current_location)) {
         return;
     }
-    const uint8_t uav_status = hal.util->get_soft_armed()? MAV_ODID_STATUS_AIRBORNE : MAV_ODID_STATUS_GROUND;
+    
+    uint8_t uav_status = hal.util->get_soft_armed()? MAV_ODID_STATUS_AIRBORNE : MAV_ODID_STATUS_GROUND;
+#if HAL_PARACHUTE_ENABLED
+    // set emergency status if chute is released
+    const auto *parachute = AP::parachute();
+    if (parachute != nullptr && parachute->released()) {
+        uav_status = MAV_ODID_STATUS_EMERGENCY;
+    }
+#endif
+    // Not yet implemented in Ardupilot 7.0.4
```

Comment:
Can you add a comment to this code on why its being added commented out.

Ex. This should be uncommented once ... happens.

And add to that comment a TODO: AS-XXXX to track the work for when it does get done?

Reply:
will do

Reply:
N/A with merge from [Copter-4.0-ODID](https://github.com/ArduPilot/ardupilot/compare/Copter-4.0-ODID)